common/src/main/java/io/crate/types/IntervalType.java

```diff
@@ -141,6 +141,14 @@ public void writeValueTo(StreamOutput out, Period p) throws IOException {
         }
     }
 
+    public static Period add(Period a, Period b) {
```

Comment:
Can we inline those? Shallow methods like this tend to add more complexity due to the indirection. They don't encapsulate anything really.